hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/map/MapPutAllMessageTask.java

```diff
@@ -47,7 +47,7 @@ public MapPutAllMessageTask(ClientMessage clientMessage, Node node, Connection c
     protected Operation prepareOperation() {
         MapEntries mapEntries = new MapEntries(parameters.entries);
         MapOperationProvider operationProvider = getMapOperationProvider(parameters.name);
-        return operationProvider.createPutAllOperation(parameters.name, mapEntries);
+        return operationProvider.createPutAllOperation(parameters.name, mapEntries, parameters.triggerMapLoader);
```

Comment:
After you regenerate the codec with correct since version. You will see an extra field `parameters.isTriggerMapLoaderExists` this should if the client is aware of the new field or not. 
if `isTriggerMapLoaderExists` is `true` then your field has two options as you did. But if it is `false` then, you should fall back to the old behavior and pass a `true` value into the `triggerMapLoader` parameter of the operation.

Reply:
I have updated.